wp/resources/sample_binaries/conditional_call/main.asm

```diff
@@ -0,0 +1,15 @@
+        global __assert_fail:function (__assert_fail.end - __assert_fail)
```

Comment:
Stylistically, I don't think these don't need to be indented. 
Otherwise, this is a great, simple test-case. Well done!

Reply:
Don't think they don't need to be indented? So they do need to be indented?

Reply:
Edit: I don't think these need to be indented. Apologies.